SA3/qa_web/static/css/bootstrap.min.css

```diff
@@ -0,0 +1,6 @@
+/*!
```

Comment:
If we can avoid adding this I think it would be best, we already link to the bootstrap cdn in base.html

Reply:
It seems that if we don't leave it there are many problems that appear on other pages. On `answerspage.html` the font we use no longer shows for the question title, and my markdown <code> does not show the grey rectangular background. As most styles work right now, I think we should solve the css stylesheets loading in another issue.

There certainly has to be something wrong with the bootstrap cdn not loading correctly.

Reply:
In your Front end change from #55 the call for this file in static is done in `base.html` but this bootstrap.min.css was not present in our static/css files.